release-notes/release-notes503.md

```diff
@@ -7,7 +7,7 @@ keywords: releasenotes newfeatures deleted technologypreview bugfixes knownissue
 
 ## Overview
 
-The Release Notes provide high-level coverage of the improvements and additions that have been implemented in TinyMCE 5.0.3 and document known problems in this release, as well as notable bug fixes, deprecated functionality, and other details.
+{{site.productname}} 5.0.3 was released for {{site.enterpriseversion}} and {{site.cloudname}} on Monday, April 15<sup>th</sup>, 2019. It includes {{site.productname}} 5.0.3 and additional changes to premium plugins. The Release Notes provide high-level coverage of the improvements and additions that have been implemented in TinyMCE 5.0.3 and document known problems in this release, as well as notable bug fixes, deprecated functionality, and other details.
```

Comment:
Another incorrect date here, this was released Thursday, 28th March, 2019. The following updates were premium plugin hotfixes.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1779/commits/bb176495886196c47aa96ca142dc929a0bd67fae